src/app/pages/select-network/components/network-list-item.layout.tsx

```diff
@@ -58,6 +71,18 @@ export function NetworkListItemLayout(props: NetworkListItemLayoutProps) {
           <Caption>{getUrlHostname(network.chain.stacks.url)}</Caption>
         </Stack>
         <NetworkStatusIndicator isActive={isActive} isOnline={isOnline} />
+        {isCustom && (
+          <Button
+            mode="tertiary"
+            ml="base"
+            onClick={e => {
+              e.stopPropagation();
```

Comment:
Curious, why not use `type="button"` here?

Reply:
I was asking here bc doesn't that allow you to remove `e.stopPropagation()`? I was thinking that was to prevent the submit behavior?